packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -534,10 +534,52 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
     /// @notice Returns the required bond for a given move kind.
     /// @param _position The position of the bonded interaction.
     /// @return requiredBond_ The required ETH bond for the given move, in wei.
-    function getRequiredBond(Position _position) public pure returns (uint256 requiredBond_) {
-        // TODO(client-pod#551): For now use a non-zero bond amount to unblock functional tests.
-        _position;
-        requiredBond_ = 0.01 ether;
+    function getRequiredBond(Position _position) public view returns (uint256 requiredBond_) {
+        // Values used here are based on the Big Bonds™ simplified bonding formula.
+        // Big Bonds™ specifies the formula as:
+        // bond = max(D, I * F * C(position))
+        // Where:
+        // - D is the deterrence deposit used to deter spam. Hardcoded to 0.08 ether.
+        // - I is the opportunity cost incentive. Hardcoded to 1.0662 based on ETH2 weekly staking yields (compounded).
+        // - F is the over-collateralization factor. Hardcoded to 1.12.
+        // - C is the cost function of countering a position.
+
+        // Here we compute C, the cost function of countering a position.
+        // Values here are designed to be good enough for an initial mainnet release and should
+        // function properly until this can be replaced with a dynamic bond formula.
+        // Currently assumes a base fee of 40 gwei, which means that the cost will be less than the
+        // deterrence deposit unless the position is at the maximum game depth. Although this means
+        // we could simplify the function, we keep it as-is for clarity and to allow for future
+        // changes to the assumed base fee without needing to significantly modify the function.
+        uint256 assumedBaseFee = 40 gwei;
+        uint256 depth = _position.depth();
+        if (depth < SPLIT_DEPTH) {
+            // Cost at output root bisection game.
+            // Sum of the cost to move (158k gas) and the cost to resolve a subgame (60k gas).
+            requiredBond_ = (158000 + 60000) * assumedBaseFee;
+        } else if (depth < MAX_GAME_DEPTH) {
+            // Cost at execution game.
+            // Sum of the cost to move (158k gas) and the cost to resolve a subgame (60k gas).
+            // Also adds the cost of offchain compute, hardcoded to 0.0043 eth for now.
+            requiredBond_ = (158000 + 60000) * assumedBaseFee + 0.0043 ether;
+        } else if (depth == MAX_GAME_DEPTH) {
+            // Cost at maximum depth.
+            // Sum of the cost to step (13k gas), the cost of offchain compute (0.0043 eth), and
+            // the cost to publish the worst-case preimage of 8MB (5.12 eth).
+            requiredBond_ = 13000 * assumedBaseFee + 0.0043 ether + 5.12 ether;
```

Comment:
We may want to make there be a smoother ramp up to this large bond because of the attack where someone spams challenges at `MAX_GAME_DEPTH-1` for cheap, and then forces tons of responses which incur high capital lockup cost. So for ex a bad situation would be that honest players pay `5 eth` bonds but the attacker is spamming invalid claims with `0.01 eth` bonds (for ex). 

To mitigate this, we just need to ensure that the bond at the previous step covers the capital lockup cost of the bond at the next step. Pretty easy way to do this is by adding a scaling factor which is based on the depth.

Reply:
This makes sense to me. We probably need to rework the spec.